src/test/java/net/library/controller/UserControllerTest.java

```diff
@@ -31,34 +35,226 @@ class UserControllerTest {
     @Autowired
     private UserService service;
 
-    @Autowired
-    public UserControllerTest(UserService service) {
-        this.service = service;
+    @BeforeEach
+    void clean(){
+        service.deleteAll();
     }
 
     @Test
-    void getUsers() throws Exception {
-        var user1 = new User("Smith1", "John1", 12);
-        var user2 = new User("Smith2", "John2", 22);
-        var user3 = new User("Smith3", "John3", 32);
+    void getAllUsers() throws Exception {
+        final var username = "Alelxo";
+        final var surname = "Bur";
+        final var name = "Alex";
+        final var email = "efaf";
+        final var phoneNumber = "380679920267";
+        final var address = "assfasfd";
+        final var xCorrelationId = getUUID();
 
-        var users = new ArrayList<>();
-        users.add(user1);
-        users.add(user2);
-        users.add(user3);
+        mvc.perform(MockMvcRequestBuilders.post(GLOBAL_BASE_URI + USERS)
+                        .header(CORRELATION_ID_HEADER_NAME, xCorrelationId)
+                        .contentType("application/json")
+                        .content("{ \"username\": \"Alelxo\", \"surname\": \"Bur\", " +
```

Comment:
Use script or manual adding to fill db. You should have several records and select by filters and without it